README.md

```diff
@@ -480,3 +480,54 @@ As above.
 #### Clock offsets
 
 As above.
+
+## twochannel_string_marker.xdf
+
+Minimal example file with two streams.
+
+### Single String Marker Stream
```

Comment:
Please use the actual stream ID `3735928559` (as in the other examples).

Please also provide a section for the other stream.